nifi-nar-bundles/nifi-shopify-bundle/nifi-shopify-processors/src/main/java/org/apache/nifi/processors/shopify/GetShopify.java

```diff
@@ -78,6 +80,7 @@
         " The list of the resources with the supported parameters can be found in the additional details.")
 @CapabilityDescription("Retrieves objects from a custom Shopify store. The processor yield time must be set to the account's rate limit accordingly.")
 @DefaultSchedule(strategy = SchedulingStrategy.TIMER_DRIVEN, period = "1 min")
+@WritesAttribute(attribute = "mime.type", description = "Sets the mime type to application/json")
```

Comment:
Minor wording adjustment, MIME should be uppercased.
```suggestion
@WritesAttribute(attribute = "mime.type", description = "Sets the MIME type to application/json")
```